routes.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+
+const rootRouter = express.Router();
+const projects = require('./projects');
+const auth = require('./auth');
+
+rootRouter.get('/projects', projects);
```

Comment:
You should import and use routers with `router.use()`. Now I am not even sure how it works on double post().